### DISCUSSION TITLE
SAS code not being highlighted within code block

### DISCUSSION THREAD

**Discussion by andyjessen** [quarto-dev/quarto-cli#11004] [Q&A] [answered]
When inserting a code block within a Quarto markdown file, I am unable to get the code highlighter to recognize SAS code and highlight the keywords. I am not trying to execute the SAS code at this time. 

Example: Code block like this renders as plain text within qmd file. 
``` sas
data cars;
    set sashelp.cars;
run;
```
I am using Quarto within VS Code. Any thoughts?

**Comment by cscheid** [comment]
Moving this to a discussion.

**Comment by cscheid** [comment, marked as answer]
Syntax highlighting in Quarto is provided by Pandoc. We describe this in the docs (see https://quarto.org/docs/output-formats/pdf-basics.html#syntax-highlighting for example) but we should have a more central place where we explain this.

In any case, you can check the set of supported languages with

````
$ quarto pandoc --list-highlight-languages
...
````

You'll see that SAS isn't supported. See https://pandoc.org/chunkedhtml-demo/13-syntax-highlighting.html for how to add a custom highlighting style.
